src/main/java/entities/Post.java

```diff
@@ -9,33 +9,33 @@
  * description,tags,collaborators, deadline, and users who favourited it.
  */
 public class Post extends Postable{
-    public static int numPostsCreated = 0;
-    /**The id is equal to the number of posts created.*/
-    private int id;
+
     /**This is a list of the users who favourited this post.*/
     private List<User> favouritedUsers;
     private String title;
     private List<String> tags;
     private String collaborators;
     private LocalDate deadline;
 
+    // a list of Postable objects created as a reply to this Postable
+    private List<Postable> replies;
+
     /**
      * @param poster the User who is posting this Post. This is an attribute of the superclass Postable.
      * @param title this Post's title.
      * @param mainDesc this Post's description.
      * @param tags the tags this Post is added to.
      * @param collaborators a description of the type of the collaborators the User who posted this Posts is looking for.
      * @param deadline the date on which the Post is to be deleted from the database.
-     * @param numPostsCreated1 the number of posts created so far.
+     * @param id  unique identifier for this post
      */
     public Post(User poster, String title, String mainDesc, List<String> tags, String collaborators,
```

Comment:
Post also needs a second constructor for when we retrieve data from the csv

Reply:
done